index.js

```diff
@@ -68,7 +69,6 @@ function main(initialState, view, opts) {
             target = patch(target, patches, opts)
```

Comment:
this effectively stops mutation during diff... but what happens during diff that can mutate state?